tests/subsys/fw_info/src/main.c

```diff
@@ -33,7 +34,7 @@ void test_fw_info_invalidate(void)
 	const uint32_t zero = 0;
 	int ret;
 
-	const struct device *flash_dev = device_get_binding(PM_S0_DEV_NAME);
+	const struct device *flash_dev = DEVICE_DT_GET(DT_NODELABEL(PM_S0_DEV));
```

Comment:
suggestion, since PM_ macros now are node ids, they could be named `PM_..._NODE`, `_DEV` may me confusing

Reply:
They are actually either DTS node or DTS node label. In pm configuration developers will use the DTS node label, mostly, but the cmake, via dt_chosen, may provide DTS node instead of label (as dt_chosen, always evaluates `&label` to node path).
Maybe what partition_manager_output.py generates to pm_config.h, as _DEV  identifier, should take form:
```
#define PM_SOME_PARTITION_DEV DEVICE_DT_GET(...)
```
directly.

Reply:
> They are actually either DTS node or DTS node label. In pm configuration developers will use the DTS node label, mostly, but the cmake, via dt_chosen, may provide DTS node instead of label (as dt_chosen, always evaluates `&label` to node path). Maybe what partition_manager_output.py generates to pm_config.h, as _DEV identifier, should take form:
> 
> ```
> #define PM_SOME_PARTITION_DEV DEVICE_DT_GET(...)
> ```
> 
> directly.

Scratch that.